pkg/importer/size/size.go

```diff
@@ -0,0 +1,26 @@
+package size
```

Comment:
If we don't intend this package to be used by the importer, then maybe it should be located under `pkg/library/size` or something similar.

Reply:
I thought of that too. But the size pkg seems like a subset of `importer`, similar to golang's `path` and `path/filepath` -- I was thinking.

Reply:
Within CDI, I'd agree.  This package would be used by other controllers to predict the size of PVCs they generate though.  In this case, it becomes a controller level package.

Also, I feel that putting it  in the importer or the controller package may imply that it's a necessary component of them, when it isn't.

Reply:
That's true (_I feel that putting it in the importer or the controller package may imply that it's a necessary component of them, when it isn't._). I agree _pkg/lib/*_ is more meaningful for this type of package. I'll do that and update Makefile.